README.md

```diff
@@ -45,87 +52,105 @@ module.exports = function(config){
+#### Files required to run your tests  
+**Command line:** `[--files|-f] node_modules/a-lib/**/*.js,src/**/*.js,a.js,test/**/*.js`  
 **Config file:** `files: ['{ pattern: 'src/**/*.js', mutated: true }, '!src/**/index.js', 'test/**/*.js']`  
 **Default value:** *none*  
+**Mandatory**: yes  
 **Description:**  
-With `files` you configure all files needed to run the tests. If the test runner you use already provides the test framework (jasmine, mocha, etc),
-you should not add those files here as well. The order in this list is important, because that will be the order in which the files are loaded.  
+With `files` you specify all files needed to run your tests. If the test runner you use already provides the test framework (Jasmine, Mocha, etc.),
+you should *not* include those files here as well.  
+The files will be loaded in the other in which they are specified.
 
-When using the command line, the list can only contain a comma seperated list of globbing expressions.
-When using the config file you can fill an array with `string`s or `InputFileDescriptor` objects:
+When using the command line, the list can only contain a comma separated list of globbing expressions.  
+When using the config file you can provide an array with `string`s or `InputFileDescriptor` objects, like so:  
 
-* `string`: The globbing expression used for selecting the files needed to run the tests.
-* `InputFileDescriptor` object: `{ pattern: 'pattern', included: true, mutated: false }` :
-   * The `pattern` property is mandatory and contains the globbing expression used for selecting the files. Using `!` to ignore files is *not* supported here. 
-   * The `included` property is optional and determines whether or not this file should be loaded initially by the test-runner (default: true)
-   * The `mutated` property is optional and determines whether or not this file should be targeted for mutations (default: false)   
+* `string`: The globbing expression used for selecting the files needed to run the tests.  
+* `InputFileDescriptor` object: `{ pattern: 'pattern', included: true, mutated: false }`:  
+   * The `pattern` property is mandatory and contains the globbing expression used for selecting the files. Using `!` to ignore files is *not* supported here.  
+   * The `included` property is optional and determines whether or not this file should be loaded initially by the test-runner (default: true)  
+   * The `mutated` property is optional and determines whether or not this file should be targeted for mutations (default: false)  
 
-*Note*: To include a file/folder which start with an exclamation mark (`!`), use the `InputFileDescriptor` syntax
+*Note*: To include a file/folder which start with an exclamation mark (`!`), use the `InputFileDescriptor` syntax.  
 
-#### Files to mutate
-**Command line:** `-m src/**/*.js,a.js` or `--mutate src/**/*.js,a.js`  
+#### Source code files to mutate  
+**Command line:** `--mutate src/**/*.js,a.js` or `-m src/**/*.js,a.js`  
```

Comment:
I noticed that with the `files` argument you use a different format: `[--files|-f] node_modules/a-lib/**/*.js,src/**/*.js,a.js,test/**/*.js` . Is there a reason why you didn't use `[--mutate|-m] src/**/*.js,a.js` here?